bitrise.yml

```diff
@@ -0,0 +1,85 @@
+#
+# bitrise.yml
+#
+# Copyright (c) 2017 Netguru Sp. z o.o. All rights reserved.
+#
+
+# CLI metadata
+
+format_version: 1.3.0
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+
+# Workflow trigger map
+
+trigger_map:
+
+  - push_branch: 'develop'
+    workflow: build-and-test
+
+  - pull_request_target_branch: 'develop'
+    workflow: build-and-test
+
+# Environment configuration
+
+app:
+  envs:
+    - XCODEBUILD_PROJECT: NGRValidator.xcworkspace
+    - XCODEBUILD_OPTIONS: _BUILD_NUMBER=$BITRISE_BUILD_NUMBER
+
+# Workflow declarations
+
+workflows:
+
+  # Top level build workflows
+
+  build-and-test:
+    envs:
+      - XCODEBUILD_SCHEME: NGRValidator
+    before_run:
+      - cache-pull
+      - pods-install
+      - xcode-test
+    after_run:
+      - cache-push
+      - deploy-artifacts
+
+  # Cocoapods
+
+  pods-install:
+    steps:
+      - script:
+          inputs:
+          - content: bundle install
+      - cocoapods-install: {}
+
+  # Cache
+
+  cache-pull:
+    steps:
+      - cache-pull: {}
+
+  cache-push:
+    steps:
+      - cache-push:
+          inputs:
+            - cache_paths: |-
+                ./Pods -> ./Podfile.lock
+
+  # Xcode
+
+  xcode-test:
+    steps:
```

Comment:
macOS tests are missing (you should add this under a new JIRA task and PR, though) 🙂